src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,35 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager extends Employee {
+
+    static List<String> managers = new ArrayList<>();
```

Comment:
`static` :) Przyznam się szczerze, że jak chodziłem na to kółko 2 lata temu, to sam tak to chciałem ugryźć xD

Taka statyczna lista daje nam możliwość posiadania kontenera na WSZYSTKICH managerów. Niby fajnie, lecz czy tak to miało wyglądać? Niestety nie. Chcieliśmy, żeby to `Ceo` posiadał własną listę managerów. I też nie chcieliśmy żeby była statyczna, bo statyczna lista jest jedna i tylko jedna, niezależnie od ilości obiektów jakie stworzymy.

Załóżmy, że mamy 2-óch `Ceo`. I jeśli lista byłaby statyczna, każdy z nich dzieliłby listę managerów. A my chcemy, żeby każdy miał swoją własną listę zatrudnionych przez siebie Managerów.

Reasumując ta lista jest tutaj niepotrzebna :)

ALE!

Jednak mimo wszystko jakaś lista jest nam tutaj potrzebna.... jaka?

Lista **pracowników!**

Każdy Manager ma pod sobą pracowników. Każdy Manager zatem musi posiadać taką listę :)
`private List<Employee> employees = new ArrayList<>();`
Tak wyglądałaby jej deklaracja. Dzięki takiej liście o typie `Employee` każdy Manager ma dostęp do wszystkich swoich pracowników. Jest to niezbędne do poprawnego funkcjonowania wyświetlania hierarchii pracowników naszej firmy.